.vimrc

```diff
@@ -177,7 +224,7 @@
 
 " Vim UI {
 
-    if !exists('g:override_spf13_bundles') && filereadable(expand("~/.vim/bundle/vim-colors-solarized/colors/solarized.vim"))
+    if !exists('g:override_spf13_bundles') && filereadable(g:spf13_vim_data_home . "/bundle/vim-colors-solarized/colors/solarized.vim")
```

Comment:
I think we might want to have `g:spf13_vim_bundle_something` which defaults to `g:spf13_vim_data_home . "/bundle"`